tests/density-screen-3/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+include(TestingMacros)
+
+add_regression_test(density-screen-3 "psi;dft;scf;medlong")
+set_tests_properties(density-screen-3 PROPERTIES COST 500)
```

Comment:
Lest this test never gets run, add the name of the test to the big list in `tests/CMakeLists.txt`, please. And add a `test_input.py` file to this dir.

Reply:
Thanks, done